README.md

```diff
@@ -33,6 +33,7 @@ How to Contribute
 
 What's New
 ----------
+* [Version 1.4.0 Release](to be created - https://github.com/apache/incubator-mxnet/releases/tag/1.4.0) - MXNet 1.4.0 Release.
```

Comment:
Since there is no 1.4.0 tag yet this will be effectively a broken link until the tag will be created. Maybe this part of the change need to be postponed.